app/code/Magento/Directory/Model/Currency/Import/CurrencyConverterApi.php

```diff
@@ -143,4 +164,27 @@ protected function _convert($currencyFrom, $currencyTo)
     {
         return 1;
     }
+
+    /**
+     * Validates rates response.
+     * @param array $response
+     * @param string $baseCurrency
+     * @param string $currenciesTo
+     * @return bool
+     */
+    private function validateResponse(array $response, string $baseCurrency, string $currenciesTo): bool
```

Comment:
I would keep the naming, ie $currencyFrom and $currencyTo

Reply:
Argument name updated